easybuild/easyconfigs/a/AFNI/AFNI-linux_openmp_64-goolf-1.5.14.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+
+
+name = 'AFNI'
+version = 'linux_openmp_64'
+# see VersionDate in caps3 output
+
+easyblock = 'PackedBinary'
+
+homepage = 'http://afni.nimh.nih.gov'
+description = """Free software for analysis and display of FMRI data """
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+
+source_urls = [('http://afni.nimh.nih.gov/pub/dist/tgz/')]
+sources = ['%(version)s.tgz']
+
+dependencies = [
+    ('R', '3.1.2'),
+    ('PyQt', 'x11-gpl-4.11.3'),
+]
+
+osdependencies = [
+    ('libXp','tcsh')
+]
```

Comment:
also, the syntax here is probably not what you want... by including both `libXp` and `tcsh` in a tuple, you're stating that having either of both is sufficient... if `libXp` is there, EasyBuild will stop checking

so, the correct way would be `osdependencies = ['libXp', 'tcsh']`

but resolving these via EasyBuild would be better

a nice trick to avoid pulling in a dozen more dependencies is to use `eb --filter-deps=libXp`; that way, EasyBuild will not try to install `libXp` as a module, and simply assume it's there... kudos to @pescobar for that nifty trick. :-)